server/__tests__/llm/deepseek.test.ts

```
@@ -0,0 +1,439 @@
+  llmUsage: {
+    create: jest.fn(),
+  },
+}));
+
+const mockPost = jest.fn();
+const mockAxiosCreate = jest.fn(() => ({
+  post: mockPost,
+}));
+
+(axios as jest.Mocked<typeof axios>).create = mockAxiosCreate;
+const mockLlmUsageCreate = db.llmUsage?.create as jest.Mock;
```

ESLint

[error] 'mockLlmUsageCreate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)